nav2_navfn_planner/src/navfn_planner.cpp

```diff
@@ -544,12 +544,12 @@ NavfnPlanner::publishEndpoints(
   marker.pose.orientation = pose.orientation;
 
   // Set the scale of the marker -- 1x1x1 here means 1m on a side
-  marker.scale.x = 3.0;
-  marker.scale.y = 3.0;
-  marker.scale.z = 3.0;
+  marker.scale.x = 0.1;
+  marker.scale.y = 0.1;
+  marker.scale.z = 0.1;
 
   builtin_interfaces::msg::Duration duration;
-  duration.sec = 0;
+  duration.sec = 10;
```

Comment:
What does this do? Seems like it would make the markers go away after 10 seconds. I'm assuming 0 means leave them there forever, or until new ones are provided.  If it is the latter, wouldn't that be preferable?